ci/parse-trivy-results.py

```diff
@@ -0,0 +1,155 @@
+"""Script to parse results from Trivy scan.
+
+1) Reads in results from two files (json and table format) generated by Trivy.
+2) Creates a GitHub issue in the given repo if any vulnerabilities were found.
+3) Fails the CI build if any vulerabilities of HIGH/CRITICAL severity exist.
+
+Both the json and table Trivy results must be created prior to running
+this script. These can be generated with:
+    trivy image --format json --light --no-progress -o <trivy_output>.json
+        gcr.io/cre-tools/ts-bridge
+    trivy image --light  --no-progress -o <trivy_output>.table
+        gcr.io/cre-tools/ts-bridge
+"""
+import json
+import sys
+from absl import app
+from absl import flags
+from github import Github
+from pathlib import Path
```

Comment:
I'm used to seeing "import github" and "import pathlib" and then using the form 'github.Github' and 'pathlib.Path' when you call those methods.  That's Google internal style though I'm not sure what public style requires.

Reply:
As discussed offline, this is the style used in https://github.com/PyGithub/PyGithub, so we will keep it unchanged.